src/components/FeedList/FeedList.store.js

```diff
@@ -12,7 +12,7 @@ export const MODULE_NAME = 'FeedList'
 export const STORE_FETCH_POSTS_PARAM = `${MODULE_NAME}/STORE_FETCH_POSTS_PARAM`
 
 // actions
-export function fetchPosts ({ activePostsOnly, afterTime, beforeTime, context, filter, offset, order, search, slug, sortBy, topic, topics, types }) {
+export function fetchPosts ({ activePostsOnly, afterTime, beforeTime, collectionToFilterOut, context, filter, first, forCollection, offset, order, search, slug, sortBy, topic, topics, types }) {
```

Comment:
oooo, this is going to be a tricky merge conflict now :(   `FeedList.store` stuff has all been moved to `Stream.store` now

Reply:
im on it